sources/packages/web/src/router/AESTRoutes.ts

```diff
@@ -539,18 +539,6 @@ export const aestRoutes: Array<RouteRecordRaw> = [
           clientType: ClientIdType.AEST,
         },
       },
-      {
-        path: AppRoutes.ViewOfferingChangeRequestComplete,
-        name: AESTRoutesConst.OFFERING_CHANGE_REQUEST_COMPLETE,
-        props: true,
-        components: {
-          default: ViewOfferingChangeRequestComplete,
```

Comment:
Please remove the obsolete variable and also the Vue file(ViewOfferingChangeRequestComplete.vue) from the file system

![image](https://github.com/bcgov/SIMS/assets/54600590/ffa74409-0d66-4b72-9891-45eb71130257)

Reply:
Also deleted, thanks for catching that too